submissions/nadsatt/memory-pair-game/index.js

```diff
@@ -0,0 +1,283 @@
+                    </div>
+                    <div class="card__back game-card__back">
+                        <img class="game-card__img" src="./images/back-${i}.png" alt="kfc-product-${i}">
+                    </div>`;
+
+    card.classList.add('card', 'game-card');
+    card.setAttribute('data-back-img-id', i);
+    card.insertAdjacentHTML('beforeend', cardInnerHtml);
+
+    return card;
+};
+
+const initGame = function(){
+    resetGameCards();
+    resetCatImgWidth();
+    solvedGameCardsNumber = 0;
+
+    shuffleGameCards();
+
+    renderInfo();
+    renderWinsNumber(++winsNumber);
+    renderFailsNumber(failsNumber = 0);
+};
+
+const resetGameCards = function(){
+    const cards = gameCardsGroup.querySelectorAll('.game-card');
+
+    moveOpenedGameCardsFromCorner(cards);
+    positionOpenedGameCardsRelatively(cards);
+
+    setTimeout(closeOpenedGameCards.bind(null, ...cards), cardFlippingTime);
+};
+
+const moveOpenedGameCardsFromCorner = function(cards){
+    cards.forEach(card =>
+        card.classList.remove('first-game-card--in-corner',
+                              'second-game-card--in-corner')
+    );
+};
+
+const positionOpenedGameCardsRelatively = function(cards){
+    cards.forEach(card =>
+        card.classList.remove('first-game-card--abs-positioned',
+                              'second-game-card--abs-positioned')
+    );
+};
+
+const closeOpenedGameCards = function(...cards){
+    cards.forEach(card =>
+        card.classList.remove('card--flipped')
+    );
+};
+
+const resetCatImgWidth = function(){
+    catImg.style.width = initCatImgWidth;
+};
+
+const shuffleGameCards = function(){
+    let cardsArr = Array.from(gameCardsGroup.children);
+    cardsArr.sort(() => Math.random() - 0.5);
+
+    cardsArr.forEach((card, i) => {
+        card.dataset.gridCell = i + 1;
+        fragment.append(card);
+    });
+
+    gameCardsGroup.append(fragment);
+};
+
+const renderInfo = function(){
+    if(infoMessageIndex === infoMessages.length) infoMessageIndex = 0;
+    infoSpan.textContent = infoMessages[infoMessageIndex++];
+};
+
+const renderWinsNumber = function(winsNumber){
+    winsNumberSpan.textContent = winsNumber;
+};
+
+const renderFailsNumber = function(failsNumber){
+    failsNumberSpan.textContent = failsNumber;
+};
+
+const handleGameCardsGroupClick = function({target: {parentElement: {parentElement: el}}}){
+    if(el.classList.contains('game-card') && !twoGameCardsOpened()){
+        const card = el;
+
+        if(!firstGameCard){
+            openGameCard(card);
+            firstGameCard = card;
+        }
+        else if(firstGameCard !== card){
+            openGameCard(card);
+            secondGameCard = card;
+            handleOpeningSecondGameCard();
+        }
+    }
+};
+
+const twoGameCardsOpened = function(){
+    return openedGameCardsNumber === 2;
+};
+
+const openGameCard = function(card){
+    card.classList.add('card--flipped');
```

Comment:
You can move this class name string to constants. I met this class name quite often in your code. It would be easier to change it in one place.